source/PlayerInfo.cpp

```diff
@@ -1137,14 +1137,14 @@ void PlayerInfo::Land(UI *ui)
 		ship->UnloadBays();
 	
 	// Ships that are landed with you on the planet should fully recharge
-	// and pool all their cargo together. Those in remote systems restore
-	// what they can without landing.
+	// and pool all their cargo together. Those that are in remote systems
+	// or cannot land on your planet restore what they can without landing.
```

Comment:
What if there is another friendly planet in the system that they can land on? Wouldn't we expect our friendly ships to land there?
(Outfitters also work for ships that are parked on a different planet but in the same system.)

We can keep this for a follow-up PR.